src/cloud-resources/src/crd/materialize.rs

```diff
@@ -49,6 +49,10 @@ pub mod v1alpha1 {
         pub environmentd_cpu_allocation: Option<String>,
         // Value for the memory request and limit for the environmentd pod
         pub environmentd_memory_allocation: Option<String>,
+        // Value for the cpu request and limit for the balancerd pod
```

Comment:
This should probably only be the request, or should be a more complex type allowing setting of both requests and limits. The CPU limits we install in SaaS do not make sense in the context of self-hosted, where they probably want to allow bursting above the request.

Reply:
yeah, paul mentioned this earlier too - i think this is fine for now just to get things working, but we should definitely start thinking about the real api for this